errors/src/main/java/com/palantir/conjure/java/api/errors/QosException.java

```diff
@@ -184,14 +254,24 @@ public List<Arg<?>> getArgs() {
 
     /** See {@link #unavailable}. */
     public static final class Unavailable extends QosException implements SafeLoggable {
+        public static final String DEFAULT_REASON = "qos-unavailable";
+
         private static final String SERVER_UNAVAILABLE = "Server unavailable";
 
         private Unavailable() {
-            super(SERVER_UNAVAILABLE);
+            super(SERVER_UNAVAILABLE, QosReason.of(DEFAULT_REASON));
```

Comment:
No reason to allocate new QosReason wrappers around the default reason string for each invocation, we can define the default QosReason at the top level in this class